### ISSUE TITLE
Landing page: Information missing

### ISSUE THREAD

**Issue by FionaDyalog** [Dyalog/documentation#44]
Information that ought to be on the Welcome page (landing page) has not yet been added. For example, see https://help.dyalog.com/19.0/#MiscPages/HelpWelcome.htm. There should also be informaiton here on how to report or fix an issue in this documentaiton set as well as in the PDFs and documents not included in this documentation set.

**Comment by xpqz** [comment]
I am not convinced all of this info belongs on the main landing page -- its purpose should be to draw people in, present journeys and be a shop front. The old page is also curiously misleading in places -- "Documentation is available in two formats: Help systems and PDF files." (written on the web, which is neither). This makes it sound like the online documentation is somewhat of an afterthought. I suggest that we place some version of this in a separate page available in the left nav instead.

**Comment by FionaDyalog** [comment]
Agreed,, this is something we need to discuss. Some of the information on the current page needs to be on the landing page, some on a different page, and some other information needs to be included that is not on the current landing page. Reowrking is required too.

**Comment by FionaDyalog** [comment]
Also see #147

**Comment by FionaDyalog** [comment]
Superseded by #498
